<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldManagerTab</class>
 <widget class="QWidget" name="WorldManagerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="WorldSubTabs">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>World Files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Folder containing levels:</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="toolTip">
             <string>The folder containing your .world files or V-Engine levels.</string>
            </property>
            <property name="whatsThis">
             <string>The folder containing your .world files or V-Engine levels.</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonSelectFolder">
            <property name="toolTip">
             <string>Opens a dialog allowing you to select a folder.</string>
            </property>
            <property name="whatsThis">
             <string>Opens a dialog allowing you to select a folder.</string>
            </property>
            <property name="text">
             <string>Select &amp;folder</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/EntityEditor_QT.qrc">
              <normaloff>:/EntityEditor/WDBFolder</normaloff>:/EntityEditor/WDBFolder</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonRefreshList">
            <property name="toolTip">
             <string>Refreshes the list of levels. Use this if you added a new level after the editor started and it doesn't show up yet.</string>
            </property>
            <property name="whatsThis">
             <string>Refreshes the list of levels. Use this if you added a new level after the editor started and it doesn't show up yet.</string>
            </property>
            <property name="text">
             <string>Refres&amp;h</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/EntityEditor_QT.qrc">
              <normaloff>:/EntityEditor/WDBRefresh</normaloff>:/EntityEditor/WDBRefresh</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonNew">
            <property name="toolTip">
             <string>Creates a new world database.</string>
            </property>
            <property name="whatsThis">
             <string>Creates a new world database.</string>
            </property>
            <property name="text">
             <string>&amp;New</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/EntityEditor_QT.qrc">
              <normaloff>:/EntityEditor/WDBAdd</normaloff>:/EntityEditor/WDBAdd</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOpen">
            <property name="toolTip">
             <string>Attempts to load the world database that matches the selected level.</string>
            </property>
            <property name="whatsThis">
             <string>Attempts to load the world database that matches the selected level.</string>
            </property>
            <property name="text">
             <string>&amp;Open</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/EntityEditor_QT.qrc">
              <normaloff>:/EntityEditor/WDBGo</normaloff>:/EntityEditor/WDBGo</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="toolTip">
             <string>Removes the world database accompanying the selected level.</string>
            </property>
            <property name="whatsThis">
             <string>Removes the world database accompanying the selected level.</string>
            </property>
            <property name="text">
             <string>D&amp;elete</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/EntityEditor_QT.qrc">
              <normaloff>:/EntityEditor/WDBDel</normaloff>:/EntityEditor/WDBDel</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QListWidget" name="listWidget">
         <property name="toolTip">
          <string>List of detected V-Engine level (.world) files.</string>
         </property>
         <property name="whatsThis">
          <string>List of detected V-Engine level (.world) files.</string>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;World files:</string>
         </property>
         <property name="buddy">
          <cstring>listWidget</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Plugin information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelPluginName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPluginVersion">
            <property name="text">
             <string>Version</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPluginDate">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>WorldSubTabs</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>toolButtonSelectFolder</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButtonRefreshList</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonOpen</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/EntityEditor_QT.qrc"/>
 </resources>
 <connections/>
</ui>
